constants.py

```diff
@@ -3,8 +3,20 @@
 """
 
 import os
+import thousandaire.trading_cost
 
 ROOT_DIR = os.path.dirname(os.path.abspath(__file__))
+
 DATA_DIR = os.path.join(ROOT_DIR, 'data')
 DATA_LIST_ALL = ['currency_price_tw', 'workdays']
+TRADING_INSTRUMENTS = {
+    ('currency', 'TW'):
+        ('USD', 'JPY', 'AUD', 'SEK', 'NZD', 'EUR', 'ZAR', 'GBP')
+}
+TRADING_CONFIGS = {
+    ('currency', 'TW'): {
+        'price_dataset': 'currency_price_tw',
+        'cost_object': thousandaire.currency_cost.CurrencyCost}
```

Comment:
I think, at least for now, it is still better to put all cost calculation objects in a single file named trading_cost.
Given that trading cost calculation is based on market, I don't expect to have many cost objects, so I feel comfortable to put them all in a file.
If we want to extend many cost objects someday, we can create a folder exactly named `trading_cost` and make separate cost objects into many files. The architecture change will not affect other parts of the project.